src/router/index.js

```diff
@@ -117,13 +117,13 @@ const routes = [
   {
     path: "/settings/products",
     name: "Producten overview",
-    component: () => import("@/views/settings/Products"),
+    component: () => import("@/views/settings/products/index"),
```

Comment:
/index is normaal gezien nooit nodig
```suggestion
    component: () => import("@/views/settings/products"),
```